common/commands/src/validator/cosmwasm/init_contract.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::context::SigningClient;
+use clap::Parser;
+use log::info;
+use mixnet_contract_common::InstantiateMsg;
+use network_defaults::NymNetworkDetails;
+use validator_client::nymd::cosmwasm_client::types::{ContractCodeId, InstantiateOptions};
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    #[clap(long)]
+    pub code_id: ContractCodeId,
+
+    #[clap(long)]
+    pub memo: Option<String>,
+
+    #[clap(long)]
+    pub label: Option<String>,
+
+    #[clap(long)]
+    pub init_message: Option<String>,
```

Comment:
Here we could use extra arguments for `funds` and optional `admin`

Reply:
Added both